setup.py

```diff
@@ -22,7 +22,7 @@ def read(*filenames, **kwargs):
 
 setup(
     name='pyrk',
-    version=get_git_version(),
+    #version=get_git_version(),
```

Comment:
I commented it because it failed. But I can't try now, because when I type 'python setup.py install', it gives an error about an old file. And I don't understand how this python setup and egg files work. The files in cd /usr/local/lib/python2.7/site-packages/pyrk-0.0.0-py2.7.egg/pyrk/ are really old.
<img width="848" alt="screen shot 2015-10-13 at 13 52 00" src="https://cloud.githubusercontent.com/assets/6147797/10468114/224ae0fc-71b2-11e5-8d3e-4e1101b28455.png">

<img width="1156" alt="screen shot 2015-10-13 at 13 49 24" src="https://cloud.githubusercontent.com/assets/6147797/10468056/ca18db3c-71b1-11e5-8584-16557d9ccb67.png">